jax/_src/scipy/stats/multinomial.py

```diff
@@ -0,0 +1,28 @@
+import scipy.stats as osp_stats
+from jax import lax
+from jax._src.lax.lax import _const as _lax_const
+from jax._src.numpy import lax_numpy as jnp
+from jax._src.numpy.lax_numpy import _promote_args_inexact
+from jax._src.numpy.util import _wraps
+from jax._src.scipy.special import gammaln, xlogy
+
+def _check_args(x, n, p):
+    if jnp.sum(x) != n:
+        raise ValueError("The sum of x should equal n.")
+    if jnp.sum(p) != 1:
+        raise ValueError("The values of p must sum to 1.")
```

Comment:
These kinds of checks won't be possible for jax implementations, because the arguments may be traced. The best solution is probably to remove `_check_args` entirely.

Reply:
This makes sense. Have you any intuition on the best way in which I could still implement the checks, or should I just remove the entirely? Perhaps returning `jnp.inf` if the checks aren't satisfied. My initial thought would be to use `jnp.where` but the docs indicate that it can't currently be JIT compiled.

Reply:
Returning `NaN` for invalid values is probably the best bet. The usual way to do that is using the three-argument form of `jnp.where`, which is compatible with transformations like JIT